scss/theme.scss

```diff
@@ -41,6 +41,16 @@
   cursor: pointer;
 }
 
+.note-content-print {
+  display: none;
+}
+
+.note-content-fast {
+  display: inherit;
+  padding: 0 10px;
+  white-space: pre-wrap;
+}
```

Comment:
did you try the inverse of this situation? hide the `div` by default, add visibility when in fast mode, and on the print stylesheet?

I think we should be able to eliminate this `.note-content-print` class altogether.
that is, something like this…

```ts
 <div className={`plaintext-note-content ${
		editor === 'fast' ? 'visible' : 'hidden'
	}`}
>
```

```css
.plaintext-note-content {
	display: none;

	&.visible {
		display: inherit;
	}
}

@media print {
	.plaintext-note-content.hidden {
		display: inherit; // or "block" or whatever is required
	}
}
```

the normal caveat applies: I have no idea what I'm doing with styling. just thought this model might feel simpler if it works and in the rare chance that you didn't try it.

Reply:
This is hiding by default: https://github.com/Automattic/simplenote-electron/pull/2271/files#diff-b4aef3f86a137f3edaf5a4e2fbb2a23fR44

Then I add .note-content-fast to make it visible. I can certainly shorten to .visible though.

Reply:
>This is hiding by default

yeah sorry for being confusing in my wording. I understand that the behavior is to be hidden by default but I was referring to the nomenclature and usage of the CSS classes. we have `note-content-print` as the CSS `class` even when we're not printing and then the only CSS properties for that are `display: none` which again reads a bit surprisingly.

my suggestion was more around the idea of naming the CSS classes according to what that component is doing - showing the untransformed note contents - and then using visibility overrides to show it. does that make any more sense than what I wrote the first time?